dev/config_fusen.yaml

```diff
@@ -39,6 +39,20 @@ flat_init_share_on_github.Rmd:
   R: R/init_share_on_github.R
   tests: tests/testthat/test-init_share_on_github.R
   vignettes: vignettes/share-on-a-github-website.Rmd
+flat_my_fun.Rmd:
+  path: dev/flat_my_fun.Rmd
+  state: active
+  R: R/my_fun.R
+  tests: tests/testthat/test-my_fun.R
+  vignettes: vignettes/bobo-rmd.Rmd
+  inflate:
+    pkg: /home/yohann/opensource/fusen
```

Comment:
As stated in the comments above, I'd like the config file to only store relative paths, so that we can collaborate on the same project without having a git diff on this file each time we change the user.